pkg/kubectl/cmd/util/factory.go

```diff
@@ -346,6 +351,15 @@ func NewFactory(optionalClientConfig clientcmd.ClientConfig) *Factory {
 		},
 		Describer: func(mapping *meta.RESTMapping) (kubectl.Describer, error) {
 			mappingVersion := mapping.GroupVersionKind.GroupVersion()
+			if mapping.GroupVersionKind.Group == federation.GroupName {
+				fedClientSet, err := clients.FederationClientSetForVersion(&mappingVersion)
+				if err != nil {
+					return nil, err
+				}
+				if mapping.GroupVersionKind.Kind == "Cluster" {
+					return &kubectl.ClusterDescriber{fedClientSet}, nil
```

Comment:
You need to change this to `&kubectl.ClusterDescriber{Interface: fedClientSet}`
hack/verify-govet is failing with:
```
pkg/kubectl/cmd/util/factory.go:360: k8s.io/kubernetes/pkg/kubectl.ClusterDescriber composite literal uses unkeyed fields
```